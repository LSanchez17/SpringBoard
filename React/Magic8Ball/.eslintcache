[{"C:\\Users\\sluis\\documents\\springboard\\springgit\\springboard\\react\\magic8ball\\src\\index.js":"1","C:\\Users\\sluis\\documents\\springboard\\springgit\\springboard\\react\\magic8ball\\src\\reportWebVitals.js":"2","C:\\Users\\sluis\\documents\\springboard\\springgit\\springboard\\react\\magic8ball\\src\\App.js":"3","C:\\Users\\sluis\\documents\\springboard\\springgit\\springboard\\react\\magic8ball\\src\\EightBall.js":"4"},{"size":500,"mtime":499162500000,"results":"5","hashOfConfig":"6"},{"size":362,"mtime":499162500000,"results":"7","hashOfConfig":"6"},{"size":1356,"mtime":1607076733970,"results":"8","hashOfConfig":"6"},{"size":868,"mtime":1607077972938,"results":"9","hashOfConfig":"6"},{"filePath":"10","messages":"11","errorCount":0,"warningCount":0,"fixableErrorCount":0,"fixableWarningCount":0},"r3sxkn",{"filePath":"12","messages":"13","errorCount":0,"warningCount":0,"fixableErrorCount":0,"fixableWarningCount":0},{"filePath":"14","messages":"15","errorCount":0,"warningCount":0,"fixableErrorCount":0,"fixableWarningCount":0},{"filePath":"16","messages":"17","errorCount":0,"warningCount":1,"fixableErrorCount":0,"fixableWarningCount":0,"source":null},"C:\\Users\\sluis\\documents\\springboard\\springgit\\springboard\\react\\magic8ball\\src\\index.js",[],"C:\\Users\\sluis\\documents\\springboard\\springgit\\springboard\\react\\magic8ball\\src\\reportWebVitals.js",[],"C:\\Users\\sluis\\documents\\springboard\\springgit\\springboard\\react\\magic8ball\\src\\App.js",[],"C:\\Users\\sluis\\documents\\springboard\\springgit\\springboard\\react\\magic8ball\\src\\EightBall.js",["18"],{"ruleId":"19","severity":1,"message":"20","line":2,"column":8,"nodeType":"21","messageId":"22","endLine":2,"endColumn":14},"no-unused-vars","'styles' is defined but never used.","Identifier","unusedVar"]