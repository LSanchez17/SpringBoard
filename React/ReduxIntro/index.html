<!DOCTYPE html>
<html lang="en">
<head>
    <meta charset="UTF-8">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <meta http-equiv="X-UA-Compatible" content="ie=edge">
    <title>Redux Demo</title>
</head>
<body>
    <div id='emojiDisplay'></div>

    <div>
        <button id='buttons'>Happy</button>
        <button id='buttons'>Sad</button>
        <button id='buttons'>Angry</button>
        <button id='buttons'>Confused</button>
    </div>


    <script src="https://cdnjs.cloudflare.com/ajax/libs/redux/4.0.1/redux.js"></script>
    <script src='./logic.js'></script>
</body>
</html>