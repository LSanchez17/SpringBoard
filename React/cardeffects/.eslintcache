[{"C:\\Users\\sluis\\documents\\springboard\\springgit\\springboard\\react\\cardeffects\\src\\index.js":"1","C:\\Users\\sluis\\documents\\springboard\\springgit\\springboard\\react\\cardeffects\\src\\reportWebVitals.js":"2","C:\\Users\\sluis\\documents\\springboard\\springgit\\springboard\\react\\cardeffects\\src\\App.js":"3","C:\\Users\\sluis\\documents\\springboard\\springgit\\springboard\\react\\cardeffects\\src\\Deck.js":"4","C:\\Users\\sluis\\documents\\springboard\\springgit\\springboard\\react\\cardeffects\\src\\Card.js":"5"},{"size":500,"mtime":499162500000,"results":"6","hashOfConfig":"7"},{"size":362,"mtime":499162500000,"results":"8","hashOfConfig":"7"},{"size":257,"mtime":1607476230388,"results":"9","hashOfConfig":"7"},{"size":2063,"mtime":1607487262334,"results":"10","hashOfConfig":"7"},{"size":243,"mtime":1607486977661,"results":"11","hashOfConfig":"7"},{"filePath":"12","messages":"13","errorCount":0,"warningCount":0,"fixableErrorCount":0,"fixableWarningCount":0,"usedDeprecatedRules":"14"},"1bw2gdt",{"filePath":"15","messages":"16","errorCount":0,"warningCount":0,"fixableErrorCount":0,"fixableWarningCount":0,"usedDeprecatedRules":"14"},{"filePath":"17","messages":"18","errorCount":0,"warningCount":0,"fixableErrorCount":0,"fixableWarningCount":0,"usedDeprecatedRules":"14"},{"filePath":"19","messages":"20","errorCount":0,"warningCount":1,"fixableErrorCount":0,"fixableWarningCount":0,"source":null},{"filePath":"21","messages":"22","errorCount":0,"warningCount":0,"fixableErrorCount":0,"fixableWarningCount":0},"C:\\Users\\sluis\\documents\\springboard\\springgit\\springboard\\react\\cardeffects\\src\\index.js",[],["23","24"],"C:\\Users\\sluis\\documents\\springboard\\springgit\\springboard\\react\\cardeffects\\src\\reportWebVitals.js",[],"C:\\Users\\sluis\\documents\\springboard\\springgit\\springboard\\react\\cardeffects\\src\\App.js",[],"C:\\Users\\sluis\\documents\\springboard\\springgit\\springboard\\react\\cardeffects\\src\\Deck.js",["25"],"C:\\Users\\sluis\\documents\\springboard\\springgit\\springboard\\react\\cardeffects\\src\\Card.js",[],{"ruleId":"26","replacedBy":"27"},{"ruleId":"28","replacedBy":"29"},{"ruleId":"30","severity":1,"message":"31","line":43,"column":8,"nodeType":"32","endLine":43,"endColumn":23,"suggestions":"33"},"no-native-reassign",["34"],"no-negated-in-lhs",["35"],"react-hooks/exhaustive-deps","React Hook useEffect has a missing dependency: 'deck.deck_id'. Either include it or remove the dependency array.","ArrayExpression",["36"],"no-global-assign","no-unsafe-negation",{"desc":"37","fix":"38"},"Update the dependencies array to be: [deck.deck_id, drew, setDraw]",{"range":"39","text":"40"},[1292,1307],"[deck.deck_id, drew, setDraw]"]