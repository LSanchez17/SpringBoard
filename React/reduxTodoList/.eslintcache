[{"C:\\Users\\sluis\\documents\\springboard\\springgit\\springboard\\react\\reduxmiddle\\src\\index.js":"1","C:\\Users\\sluis\\documents\\springboard\\springgit\\springboard\\react\\reduxmiddle\\src\\App.js":"2","C:\\Users\\sluis\\documents\\springboard\\springgit\\springboard\\react\\reduxmiddle\\src\\rootReducer.js":"3","C:\\Users\\sluis\\documents\\springboard\\springgit\\springboard\\react\\reduxmiddle\\src\\store.js":"4","C:\\Users\\sluis\\documents\\springboard\\springgit\\springboard\\react\\reduxmiddle\\src\\Todos.js":"5","C:\\Users\\sluis\\documents\\springboard\\springgit\\springboard\\react\\reduxmiddle\\src\\ListTodos.js":"6","C:\\Users\\sluis\\documents\\springboard\\springgit\\springboard\\react\\reduxmiddle\\src\\AddTodos.js":"7"},{"size":310,"mtime":1611160219545,"results":"8","hashOfConfig":"9"},{"size":160,"mtime":1611160510867,"results":"10","hashOfConfig":"9"},{"size":427,"mtime":1611203257428,"results":"11","hashOfConfig":"9"},{"size":144,"mtime":1611160167216,"results":"12","hashOfConfig":"9"},{"size":820,"mtime":1611202966221,"results":"13","hashOfConfig":"9"},{"size":191,"mtime":1611203314619,"results":"14","hashOfConfig":"9"},{"size":771,"mtime":1611164182005,"results":"15","hashOfConfig":"9"},{"filePath":"16","messages":"17","errorCount":0,"warningCount":0,"fixableErrorCount":0,"fixableWarningCount":0,"usedDeprecatedRules":"18"},"fksrim",{"filePath":"19","messages":"20","errorCount":0,"warningCount":0,"fixableErrorCount":0,"fixableWarningCount":0,"usedDeprecatedRules":"18"},{"filePath":"21","messages":"22","errorCount":0,"warningCount":0,"fixableErrorCount":0,"fixableWarningCount":0},{"filePath":"23","messages":"24","errorCount":0,"warningCount":0,"fixableErrorCount":0,"fixableWarningCount":0,"usedDeprecatedRules":"18"},{"filePath":"25","messages":"26","errorCount":0,"warningCount":0,"fixableErrorCount":0,"fixableWarningCount":0},{"filePath":"27","messages":"28","errorCount":0,"warningCount":0,"fixableErrorCount":0,"fixableWarningCount":0},{"filePath":"29","messages":"30","errorCount":0,"warningCount":0,"fixableErrorCount":0,"fixableWarningCount":0},"C:\\Users\\sluis\\documents\\springboard\\springgit\\springboard\\react\\reduxmiddle\\src\\index.js",[],["31","32"],"C:\\Users\\sluis\\documents\\springboard\\springgit\\springboard\\react\\reduxmiddle\\src\\App.js",[],"C:\\Users\\sluis\\documents\\springboard\\springgit\\springboard\\react\\reduxmiddle\\src\\rootReducer.js",[],"C:\\Users\\sluis\\documents\\springboard\\springgit\\springboard\\react\\reduxmiddle\\src\\store.js",[],"C:\\Users\\sluis\\documents\\springboard\\springgit\\springboard\\react\\reduxmiddle\\src\\Todos.js",[],"C:\\Users\\sluis\\documents\\springboard\\springgit\\springboard\\react\\reduxmiddle\\src\\ListTodos.js",[],"C:\\Users\\sluis\\documents\\springboard\\springgit\\springboard\\react\\reduxmiddle\\src\\AddTodos.js",[],{"ruleId":"33","replacedBy":"34"},{"ruleId":"35","replacedBy":"36"},"no-native-reassign",["37"],"no-negated-in-lhs",["38"],"no-global-assign","no-unsafe-negation"]