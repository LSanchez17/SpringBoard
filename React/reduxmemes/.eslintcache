[{"C:\\Users\\sluis\\documents\\springboard\\springgit\\springboard\\react\\reduxmemes\\src\\index.js":"1","C:\\Users\\sluis\\documents\\springboard\\springgit\\springboard\\react\\reduxmemes\\src\\App.js":"2","C:\\Users\\sluis\\documents\\springboard\\springgit\\springboard\\react\\reduxmemes\\src\\rootReducer.js":"3","C:\\Users\\sluis\\documents\\springboard\\springgit\\springboard\\react\\reduxmemes\\src\\MemeForm.js":"4","C:\\Users\\sluis\\documents\\springboard\\springgit\\springboard\\react\\reduxmemes\\src\\Memes.js":"5"},{"size":400,"mtime":1611282432490,"results":"6","hashOfConfig":"7"},{"size":895,"mtime":1611285973833,"results":"8","hashOfConfig":"7"},{"size":427,"mtime":1611282623180,"results":"9","hashOfConfig":"7"},{"size":1353,"mtime":1611285880243,"results":"10","hashOfConfig":"7"},{"size":679,"mtime":1611286124519,"results":"11","hashOfConfig":"7"},{"filePath":"12","messages":"13","errorCount":0,"warningCount":0,"fixableErrorCount":0,"fixableWarningCount":0,"usedDeprecatedRules":"14"},"1t2ts9o",{"filePath":"15","messages":"16","errorCount":0,"warningCount":0,"fixableErrorCount":0,"fixableWarningCount":0},{"filePath":"17","messages":"18","errorCount":0,"warningCount":0,"fixableErrorCount":0,"fixableWarningCount":0,"usedDeprecatedRules":"14"},{"filePath":"19","messages":"20","errorCount":0,"warningCount":0,"fixableErrorCount":0,"fixableWarningCount":0},{"filePath":"21","messages":"22","errorCount":0,"warningCount":1,"fixableErrorCount":0,"fixableWarningCount":0,"source":null},"C:\\Users\\sluis\\documents\\springboard\\springgit\\springboard\\react\\reduxmemes\\src\\index.js",[],["23","24"],"C:\\Users\\sluis\\documents\\springboard\\springgit\\springboard\\react\\reduxmemes\\src\\App.js",[],"C:\\Users\\sluis\\documents\\springboard\\springgit\\springboard\\react\\reduxmemes\\src\\rootReducer.js",[],"C:\\Users\\sluis\\documents\\springboard\\springgit\\springboard\\react\\reduxmemes\\src\\MemeForm.js",[],"C:\\Users\\sluis\\documents\\springboard\\springgit\\springboard\\react\\reduxmemes\\src\\Memes.js",["25"],{"ruleId":"26","replacedBy":"27"},{"ruleId":"28","replacedBy":"29"},{"ruleId":"30","severity":1,"message":"31","line":15,"column":17,"nodeType":"32","endLine":15,"endColumn":71},"no-native-reassign",["33"],"no-negated-in-lhs",["34"],"jsx-a11y/img-redundant-alt","Redundant alt attribute. Screen-readers already announce `img` tags as an image. You donâ€™t need to use the words `image`, `photo,` or `picture` (or any specified custom words) in the alt prop.","JSXOpeningElement","no-global-assign","no-unsafe-negation"]