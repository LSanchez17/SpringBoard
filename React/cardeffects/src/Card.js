import React from 'react'

const Card = ({image, value, suit}) => {
    console.log('image', image)
    console.log('value', value)
    console.log('suit', suit);
    return(
        <div>
            <h3>{value} of {suit}</h3>
            <img src={image} alt={value + suit}></img>
        </div>
    )
}

export default Card;