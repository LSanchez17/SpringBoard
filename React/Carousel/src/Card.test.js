import React from "react";
import { render, fireEvent } from "@testing-library/react";
import Card from "./Card";

it('renders card component', () => {
  render(<Card />);
})

it('Should pass snapshot', () => {
  const {aFragment} = render(<Card />);
  expect(aFragment).toMatchSnapshot();
})
