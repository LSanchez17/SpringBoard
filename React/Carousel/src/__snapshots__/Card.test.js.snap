// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should pass snapshot 1`] = `undefined`;
